static/app/views/monitors/components/monitorForm.tsx

```diff
@@ -202,130 +221,142 @@ function MonitorForm({
+                      options={timezoneOptions}
+                      required
+                      stacked
+                      inline={false}
+                    />
+                  </ScheduleGroupInputs>
+                );
               }
+              if (schedule_type === 'interval') {
+                return (
+                  <ScheduleGroupInputs>
+                    <StyledNumberField
+                      name="config.schedule.frequency"
+                      placeholder="e.g. 1"
+                      required
+                      stacked
+                      inline={false}
+                    />
+                    <StyledSelectField
+                      name="config.schedule.interval"
+                      options={getIntervals(
+                        Number(form.current.getValue('config.schedule.frequency') ?? 1)
+                      )}
+                      required
+                      stacked
+                      inline={false}
+                    />
+                  </ScheduleGroupInputs>
+                );
+              }
+              return null;
             }}
           </Observer>
-        </PanelBody>
-      </Panel>
+        </InputGroup>
+        <StyledListItem>{t('Set a missed status')}</StyledListItem>
+        <ListItemSubText>
+          {t('How long to wait before we consider a check-in as missed.')}
+        </ListItemSubText>
+        <InputGroup>
+          <StyledNumberField
+            name="config.checkin_margin"
+            placeholder="e.g. 30"
+            stacked
+            inline={false}
+          />
+        </InputGroup>
+        <StyledListItem>{t('Set a failed status')}</StyledListItem>
+        <ListItemSubText>
+          {t(
+            "How long a check-in is allowed to run before it's considered failed. If the job encounters an error it will also fail."
+          )}
+        </ListItemSubText>
+        <InputGroup>
+          <StyledNumberField
+            name="config.max_runtime"
+            placeholder="e.g. 30"
+            stacked
+            inline={false}
+          />
+        </InputGroup>
+      </StyledList>
     </Form>
   );
 }
 
 export default MonitorForm;
 
-const CombinedField = styled('div')`
-  display: grid;
-  grid-template-columns: 50% 1fr 1fr;
-  align-items: center;
-  border-bottom: 1px solid ${p => p.theme.innerBorder};
+const StyledList = styled(List)`
+  width: 700px;
+`;
+
+const StyledTextCopyInput = styled(TextCopyInput)`
+  padding: 0;
 `;
 
 const StyledNumberField = styled(NumberField)`
   padding: 0;
-  border-bottom: none;
 `;
 
 const StyledSelectField = styled(SelectField)`
-  padding-left: 0;
+  padding: 0;
+`;
+
+const StyledFieldGroup = styled(FieldGroup)`
+  padding: 0;
+`;
+
+const StyledTextField = styled(TextField)`
+  padding: 0;
+`;
+
+const StyledSentryProjectSelectorField = styled(SentryProjectSelectorField)`
+  padding: 0;
+`;
+
+const StyledListItem = styled(ListItem)`
```

Comment:
Do we have to change this? How do we do it in alerts?

Reply:
copied from alerts haha

Reply:
https://github.com/getsentry/sentry/blob/952e4de9a068694babaf795c2949fedbdd1ba606/static/app/views/alerts/rules/metric/ruleConditionsForm.tsx#L564

Reply:
@scttcper should we extract this out or something?